symfony/src/Twig/AppExtension.php

```diff
@@ -35,9 +38,9 @@ public function getFunctions()
             new TwigFunction('slugify', [$this, 'slugify']),
             new TwigFunction('timezone_display', [$this, 'timezoneDisplay']),
             new TwigFunction('codebook_display', [$this, 'getCodeBookDisplay']),
-            new TwigFunction('organization_display', [$this, 'getAwardeeDisplay']),
             new TwigFunction('awardee_display', [$this, 'getAwardeeDisplay']),
```

Comment:
I think we can flag a few of these as deprecated since we're getting `SiteService` loaded in (help reduce redundant code).